ui/app/models/azure/config.js

```diff
@@ -65,4 +65,41 @@ export default class AzureConfig extends Model {
   get formFields() {
     return expandAttributeMeta(this, this.configurableParams);
   }
+
+  // "filedGroupsWif" and "fieldGroupsAzure" are passed to the FormFieldGroups component to determine which group to show in the form (ex: @groupName="fieldGroupsWif")
+  get fieldGroupsWif() {
```

Comment:
This seems like quite a bit of extra code (these two getters plus the `formFieldGroups` function below) - I think we already have helpers that do this 🤔  Can we reuse any of the model decorators or existing patterns to get the same functionality?

Reply:
Did a little updating, let me know if that helps.